.circleci/config.yml

```diff
@@ -1,32 +0,0 @@
-# Javascript Node CircleCI 2.0 configuration file
-#
-# Check https://circleci.com/docs/2.0/language-javascript/ for more details
-#
-version: 2
-jobs:
-  build:
-    docker:
-      # specify the version you desire here
-      # https://circleci.com/docs/2.0/circleci-images/#nodejs
-      - image: circleci/node:current
-
-    working_directory: ~/repo
-
-    steps:
-      - checkout
-
-      - run:
-          name: Install Dependencies
-          command: yarn install
-
-      - run:
-          name: Compiles
-          command: npm run build
-
-      - run:
-          name: Run linter
-          command: npm run lint
-
-      - run:
-          name: Test types are built
-          command: cd fixtures/sample-typescript-project && npm install && npm run build && npm run lint && npm run test
```

Comment:
This step isn't present in the new GitHub Actions workflow. Is that intentional? I presume not, since it's removal is not mentioned in the PR description.

Reply:
it didn't seem necessary because this repo is going away.  there will be no need to validate these types going forward and the value of this test is questionable as well.